scripts/update/Updater.php

```diff
@@ -494,5 +494,7 @@ public function update($initialVersion)
         }
 
         $this->skip('12.12.0', '12.17.0');
+
+        //Updater script is deprecated. Use migration scripts.
```

Comment:
BTW migrations are part of `tao` and if we use it in `generis` it will introduce dependency on tao.